pkgs/servers/apache-kafka/default.nix

```diff
@@ -83,5 +83,7 @@ stdenv.mkDerivation rec {
     maintainers = [ maintainers.ragge ];
     platforms = platforms.unix;
   };
-  passthru = { inherit jdk17_headless; };
+  passthru = {
+    inherit jre; # Used by the NixOS module to select the supported jre
+  };
```

Comment:
Please move this straight above meta.

Reply:
Why? :)

Reply:
(I don't mind, I'd just like to understand why rather than just blindly doing dark magic on your request :smile_cat:)

Reply:
meta should always be last which is also written somewhere in the contributing guide and things after it get often overlooked because of that.